protocols/mempool-overview.md

```diff
@@ -105,25 +105,30 @@ To this end, we first provide a (brief) definition of what is a ledger, that is
 At a process $p$, we shall write $p.var$ the local variable $var$ at $p$.
```

Comment:
This is just personal preference: I would change the word "process" to "node". I think in this context it makes more sense. A node has a ledger which is at some height. A node sends and receives messages.

Reply:
I prefer to stick with the term _process_, as this is the one commonly encountered in specifications (see e.g., quint ones).